### PR TITLE
UD-1051 Allow to connect to remote API

### PR DESCRIPTION
**Pull request by benoitf**
**_3 Upvotes**_ Change-Id: Id708b9458f8cfff1785b57dea8044398bcaf1fe5
Signed-off-by: Florent BENOIT fbenoit@codenvy.com

### CODE REVIEW

File: dashboard/src/components/api/che-websocket.factory.js [hunk 1]
```diff
@@ -66,6 +66,24 @@ export class CheWebsocket {
...
+   * @param workspaceId the workspaceID used as suffix for the URL
+   */
+  getRemoteBus(websocketURL, workspaceId) {
+    var currentBus = this.sockets.get(workspaceId);
+    if (!currentBus) {
+      var dataStream = this.$websocket(websocketURL + workspaceId);
+      var bus = new MessageBus(dataStream);
+      this.sockets.set(workspaceId, bus);
+      currentBus = bus;
+      currentBus = bus;
```
Comment (akurinnoy) [inline dashboard/src/components/api/che-websocket.factory.js]: Seems like duplicated lines.

Comment (benoitf) [inline dashboard/src/components/api/che-websocket.factory.js]: fixed

**Comment by akurinnoy** [comment]
+1

**Comment by ashumilova** [comment]
+1

**Comment by olexii4** [comment]
+1

**Pull request merged**